src/langsmith/deploy-self-hosted-full-platform.mdx

```diff
@@ -80,6 +80,20 @@ As of v0.12.0, the `langgraphPlatform` option is deprecated. Use `config.deploym
 
     Your self-hosted infrastructure is now ready to create deployments.
 
+### (Optional) Configuring Additional Data Planes
+In addition to the existing data plane already created in the above steps, you can create more data planes that reside in different Kubernetes clusters.
+
+1. Read through the cluster organization guide in the [hybrid deployment documentation](/langsmith/hybrid#listeners) to understand how to best organize this.
```

Comment:
Suggestion: `how to best organize this` --> `how to best organize this for your use case`.